src/flint/types/fmpz_mpoly.pyx

```diff
@@ -750,10 +773,104 @@ cdef class fmpz_mpoly(flint_mpoly):
+        return res
+
+    def term_content(self):
+        """
+        Return the GCD of the terms of ``self``. If ``self`` is zero, then the result will
+        be zero, otherwise it will be a monomial with positive coefficient.
+
+            >>> from flint import Ordering
+            >>> ctx = fmpz_mpoly_ctx.get_context(2, Ordering.lex, 'x')
+            >>> x0, x1 = ctx.gens()
+            >>> f = 3 * x0**2 * x1 + 6 * x0 * x1
+            >>> f.term_content()
+            3*x0*x1
+        """
+        cdef fmpz_mpoly res = create_fmpz_mpoly(self.ctx)
+        fmpz_mpoly_term_content(res.val, self.val, self.ctx.val)
+        return res
+
+    def resultant(self, other, var):
+        """
+        Return the resultant of ``self`` and ``other`` with respect to variable ``var``.
+
+            >>> from flint import Ordering
+            >>> ctx = fmpz_mpoly_ctx.get_context(2, Ordering.lex, 'x')
+            >>> x0, x1 = ctx.gens()
+            >>> f = x0**2 * x1 + x0 * x1
+            >>> g = x0 + x1
+            >>> f.resultant(g, 'x1')
+            x0^3 + x0^2
+        """
+        cdef:
+            fmpz_mpoly res
+            slong i
+
+        if not typecheck(other, fmpz_mpoly):
+            raise TypeError("argument must be a fmpz_mpoly")
+        elif (<fmpz_mpoly>self).ctx is not (<fmpz_mpoly>other).ctx:
+            raise IncompatibleContextError(f"{(<fmpz_mpoly>self).ctx} is not {(<fmpz_mpoly>other).ctx}")
+
+        i = self.ctx.variable_to_index(var)
+        res = create_fmpz_mpoly(self.ctx)
+        if not fmpz_mpoly_resultant(res.val, self.val, (<fmpz_mpoly>other).val, i, self.ctx.val):
+            raise RuntimeError(f"failed to compute resultant with respect to {var}")
+        return res
+
+    def discriminant(self, var):
+        """
+        Return the discriminant of ``self`` with respect to variable ``var``.
+
+            >>> from flint import Ordering
+            >>> ctx = fmpz_mpoly_ctx.get_context(2, Ordering.lex, 'x')
+            >>> x0, x1 = ctx.gens()
+            >>> f = (x0 + x1)**2 + 1
+            >>> f.discriminant('x1')
+            -4
+
+        """
+        cdef:
+            fmpz_mpoly res
+            slong i
+
+        i = self.ctx.variable_to_index(var)
+        res = create_fmpz_mpoly(self.ctx)
+        if not fmpz_mpoly_discriminant(res.val, self.val, i, self.ctx.val):
+            raise RuntimeError(f"failed to compute discriminant with respect to {var}")
+        return res
+
+    def primitive(self):
+        """
+        Return the content and primitive of ``self``.
+
+            >>> from flint import Ordering
+            >>> ctx = fmpz_mpoly_ctx.get_context(2, Ordering.lex, 'x')
+            >>> x, y = ctx.gens()
+            >>> f = 4*x + 2*x*y
+            >>> f.primitive()
+            (2, x0*x1 + 2*x0)
+        """
+        cdef fmpz content = self.content()
+        return content, self._truediv_scalar_(content, True)
```

Comment:
I think this is the only place that uses the second argument assume_exact. This could just be changed to call
```
self._truediv_exact_scalar_(content)
```
Or Perhaps we can just call it:
```
self._divexact_scalar_(content)
```
Possibly good to line up with what FLINT calls it...